### DISCUSSION TITLE
Google analytics and hotjar cookies domain is set as .vercel.app. I need to block all cookies.

### DISCUSSION THREAD

**Discussion by Sooraj-s-98** [vercel/next.js#45935] [Help]
<img width="754" alt="hy" src="https://user-images.githubusercontent.com/52352285/218977135-15726f71-206c-49e0-baa9-7282cf81f0b7.png">

**Comment by icyJoseph** [comment]
Checked on my site and they are on my top domain and not `vercel.app`, could you share how you setup google analytics and what domain are using?

**Comment by Sooraj-s-98** [comment]
I need to block all cookies from GTM and hotjar. Currently I am using CookieYes for cookie consent solution.  Cookieyes script not working when i used the default  `afterInteractive` . But `beforeInteractive` worked for me. 

what is the best way ?
